cli/slsa-verifier/verify/verify_artifact.go

```diff
@@ -54,6 +54,7 @@ func (c *VerifyArtifactCommand) Exec(ctx context.Context, artifacts []string) (*
 			ExpectedVersionedTag:   c.SourceVersionTag,
 			ExpectedTag:            c.SourceTag,
 			ExpectedWorkflowInputs: c.BuildWorkflowInputs,
+			ExpectedBuilderPath:    "https://github.com/slsa-framework/slsa-github-generator/.github/workflows/",
```

Comment:
this need not be provided here, otherwise it will become a default for all builders?

Reply:
I did this to hardcode the path for our BYOB builders to not have to insert `--builder-id` on the thought that it could later be changed to be input-based.

After our talk earlier today, I made the change we decided on to hardcode the `expectedBuilderIDPath` in `provenance.go`. 

Functionality is still the same: only BYOB builders are supported at this time for the non-compulsory BuilderID feature.